v01-object-or-null/src/main/java/dev/futa/tutorial/pesel/v01/PeselService.java

```diff
@@ -0,0 +1,114 @@
+package dev.futa.tutorial.pesel.v01;
+
+import dev.futa.tutorial.pesel.Gender;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Pattern;
+
+import static dev.futa.tutorial.pesel.Gender.FEMALE;
+import static dev.futa.tutorial.pesel.Gender.MALE;
+import static java.lang.Character.getNumericValue;
+import static java.lang.Integer.parseInt;
+
+public class PeselService {
+
+  private static final byte PESEL_LENGTH = 11;
```

Comment:
`byte` mnie nie przekonuje ;-)

Reply:
Dlaczego?

Reply:
W jakim celu użyty `byte`, żeby oszczędzić pamięć, bo wartość to tylko 11? I tak będzie wszędzie rzutowane na `int`